hikari/impl/cache.py

```diff
@@ -128,13 +155,29 @@ def _assert_has_intent(self, intents: intents_.Intents, /) -> None:
                 stacklevel=1,
             )
 
+    def _is_cache_enabled_for(self, setting: str) -> bool:
+        return bool(self._settings_map["enable"] and self._settings_map[setting])
```

Comment:
would be more consistent to use ENABLE here

Reply:
My bad :sweat_smile: